ui/raidboss/data/03-hw/alliance/weeping_city.txt

```diff
@@ -0,0 +1,416 @@
+### THE WEEPING CITY OF MHACH
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~~~~~~#
+# ARACHNE EVE #
+#~~~~~~~~~~~~~#
+
+# -ii 368 1835 1836
+# -ic "Webmaiden" "Spitting Spider" "Skittering Spider" "Earth Aether"
+
+0 "--sync--" sync /00:0839:.*Queen's Room will be sealed off/
```

Comment:
Can you make this and the other 00:0839 syncs not have a `.*` before the zone name and be the exact line? I think this can just be `00:0839:The Queen's Room will be sealed off`, with a similar "the" in front of all of the others.